### PR TITLE
Fix compatibility issues with earlier versions of Symfony

### LINKED ISSUE #16: Symfony version requirement
You should set symfony version to at least 2.5 since `Symfony\Component\Console\Question\Question` is not available in 2.4 and it's required to generate the gulpfile

### PR DESCRIPTION
**Pull request by regularjack**
Fix #16

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -60,9 +60,21 @@ matrix:
...
+      symfony/console:${SYMFONY_VERSION} \
+      symfony/config:${SYMFONY_VERSION} \
+      symfony/dependency-injection:${SYMFONY_VERSION} \
+      symfony/event-dispatcher:${SYMFONY_VERSION} \
+      symfony/http-kernel:${SYMFONY_VERSION} \
+      symfony/process:${SYMFONY_VERSION} \
+      symfony/templating:${SYMFONY_VERSION} \
+      symfony/filesystem:${SYMFONY_VERSION} \
+      symfony/http-foundation:${SYMFONY_VERSION} \
+      symfony/routing:${SYMFONY_VERSION}
```
Comment (xabbuh) [inline .travis.yml]: I would simply require `symfony/symfony` instead.

Comment (regularjack) [inline .travis.yml]: That makes sense, will do. The disadvantage to doing that is that some needed component might not be required in `composer.json` and the tests wouldn't catch it, but that's unlikely to happen.

Comment (xabbuh) [inline .travis.yml]: For this I would add a job that does not define the `$SYMFONY_VERSION` env var and in that case simply run `composer install`.

Comment (regularjack) [inline .travis.yml]: That would indeed be a solution but I think it's overkill for now, I think we can live without.

**Pull request merged**